rootfs-builder/rootfs.sh

```diff
@@ -14,6 +14,12 @@ script_name="${0##*/}"
 script_dir="$(dirname $(readlink -f $0))"
 AGENT_VERSION=${AGENT_VERSION:-}
 GO_AGENT_PKG=${GO_AGENT_PKG:-github.com/kata-containers/agent}
+RUST_AGENT_PKG=${RUST_AGENT_PKG:-github.com/kata-containers/kata-containers}
+RUST_AGENT=${RUST_AGENT:-no}
+RUST_VERSION="null"
+RUST_SRC_PATH=${RUST_SRC_PATH:-${HOME}/rust}
+CMAKE_VERSION=${CMAKE_VERSION:-"3.15.3"}
+MUSL_VERSION=${MUSL_VERSION:-"1.1.23"}
```

Comment:
Why do we need `CMAKE_VERSION` and `MUSL_VERSION` when we die if `detect_cmake_version()` or `detect_musl_version()` fail?

Reply:
Let make it "null"